src/pages/_app.tsx

```diff
@@ -29,7 +31,11 @@ const MyApp = ({
   return (
     <SessionProvider session={session}>
       <StyledEngineProvider injectFirst>
-        {getLayout(<Component {...pageProps} />)}
+        {getLayout(
+          <RootLayout>
```

Comment:
I think RootLayout should wrap the getLayout function, not the other way around

Reply:
Got it, was confused on the intention before. Will update this as well as only have the admin layout show the admin page.